CONTRIBUTING.md

```diff
@@ -86,18 +86,24 @@ While you're working away in your branch it's quite possible that your upstream/
 1. `git pull upstream master`
 1. `git checkout myBranch`
 1. `git rebase master myBranch`
-1. `git push origin master` - (optional) this this makes sure your remote master is up to date
-1. if you previously pushed your branch to your origin, you need to force push the rebased branch - `git push origin myBranch -f`
+1. `git push origin master` - (optional) this makes sure your remote master is up to date if you previously pushed your branch to your origin, you need to force push the rebased branch - `git push origin myBranch -f`

```

Comment:
This is actually not intended. I will add a new revision because of the ReSharper thing above so I can fix this as well. Ok?